File: 15-WebWorkers/language-examples/lists.js
const a = [98, 93, 77, 82, 83];
let aSum = 0;
for (value of a) {
    aSum += value;
}
console.log(aSum);
// alternative
const aSum2 = a.reduce((acc, num) => acc + num);
console.log(aSum2);

File: 14-Python/hello_world/lists.py
def main():
    a = [98, 93, 77, 82, 83]
    a_sum = 0
    for value in a:
        a_sum += value
    print(a_sum / len(a))
    # alternative
    a_sum2 = sum(a)
    print(a_sum2 / len(a))


if __name__ == '__main__':
    main()
